### DISCUSSION TITLE
flow : update using variable

### DISCUSSION THREAD

**Discussion by nervusdm** [directus/directus#14724] [Archive]
How Can I use current_user in a flow update ?

<img width="960" alt="Capture d’écran 2022-07-28 à 16 50 09" src="https://user-images.githubusercontent.com/4217180/181567154-6f6ec166-2b16-429d-94e9-4f2537dc0fca.png">

**Comment by rijkvanzanten** [comment]
Hi @nervusdm! There is no "current user" in a flow, as the flow may or may not be triggered by a user in the first place. You should be able to rely on the `$accountability` section of the flow payload. Keep in mind that this value might be `null` if the flow was triggered outside of user interaction 👍🏻
